tests/test_horizon_scores.py

```
@@ -255,6 +255,102 @@ def test_raises_exception_without_anthropic_key(self, mock_pd):
+        assert "last_run_at" in data_store.write_keys
+
+    @patch('steps.update_horizon_scores.query_tasks')
+    @patch('steps.update_horizon_scores.generate_rubric')
+    @patch('steps.update_horizon_scores.fetch_page_blocks')
+    @patch('steps.update_horizon_scores.parse_blocks_to_text')
+    @patch('steps.update_horizon_scores.fetch_page_metadata')
+    def test_failing_data_store_read_degrades_without_failing_run(
+        self, mock_meta, mock_parse, mock_blocks, mock_rubric, mock_query, mock_pd, capsys
+    ):
+        class FailingDataStore(dict):
+            def get(self, key, default=None):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `get`

Add return type annotation: `NoReturn`

(ANN202)